pkg/descheduler/node/node.go

```diff
@@ -173,6 +177,14 @@ func podFitsNodes(nodeIndexer podutil.GetPodsAssignedToNodeFunc, pod *v1.Pod, no
 	return filteredLen > 0
 }
 
+func addFitNodeNameToPod(pod *v1.Pod, node *v1.Node) {
```

Comment:
Adding a fit node to the pod's annotation is not the most appropriate best practice. It seems more reasonable to add a return value to `type FilterFunc func(*v1.Pod) bool`, changing it to `type FilterFunc func(*v1.Pod) (string, bool)`. However, many filters do not require this string return value. Is there a better implementation approach?